02_calculator/calculator_spec.rb

```diff
@@ -79,21 +79,41 @@
 
 describe "#multiply" do
 
-  it "multiplies two numbers"
-
-  it "multiplies several numbers"
+  it "multiplies two numbers" do
+      expect(multiply(2, 3)).to eq(6)
+      expect(multiply(2, -3)).to eq(-6)
+      expect(multiply(-2, -3)).to eq(6)
+      expect(multiply(2, 0)).to eq(0)
+  it "multiplies several numbers" do
+      expect(multiply(2, 3, 4)).to eq(24)
+      expect(multiply(2, -3, -4)).to eq(24)
+      expect(multiply(-2, -3, -4)).to eq(-24)
+      expect(multiply(2, 0, 4)).to eq(0)
   
 end
 
 describe "#power" do
-  it "raises one number to the power of another number"
+  it "raises one number to the power of another number" do
+      expect(power(2,3)).to eq(8)
+      expect(power(2,0)).to eq(1)
+    end
 end
 
 # http://en.wikipedia.org/wiki/Factorial
 describe "#factorial" do
-  it "computes the factorial of 0"
-  it "computes the factorial of 1"
-  it "computes the factorial of 2"
-  it "computes the factorial of 5"
-  it "computes the factorial of 10"
+  it "computes the factorial of 0" do
+      expect(factorial(0)).to eq(1)
+    end
+  it "computes the factorial of 1" do
+      expect(factorial(1)).to eq(1)
+    end
+  it "computes the factorial of 2" do
+      expect(factorial(2)).to eq(2)
+    end
+  it "computes the factorial of 5" do
+      expect(factorial(5)).to eq(`120`)
```

Comment:
The backticks make Ruby run a shell command, which you don't want here.

```suggestion
      expect(factorial(5)).to eq(120)
```